src/main/java/com/powsybl/openloadflow/network/LfNetwork.java

```diff
@@ -224,6 +237,9 @@ public void updateSlackBusesAndReferenceBus() {
                 LOGGER.info("Network {}, reference generator is {}", this, referenceGenerator.getId());
                 referenceGenerator.setReference(true);
             }
+            if (connectivity != null) {
```

Comment:
We have another place in the code with `connectivity.setMainComponentVertex(getSlackBuses().get(0));`, is it normal?

Reply:
ok, understand.